backend/src/main/java/saci/application/RoleController.java

```diff
@@ -63,4 +68,13 @@ public ResponseEntity<List<Role>> getRoles() {
         List<Role> roles = roleService.getRoles();
         return roles.isEmpty() ? ResponseEntity.noContent().build() : ResponseEntity.ok(roles);
     }
+
+
+    @PutMapping("/{roleId}")
+    public ResponseEntity<Role> editRole(@PathVariable Long roleId, @RequestBody Role updatedRole) {
```

Comment:
Please check if the updatedRole is valid, using @Valid, @plinionr3 can also help you with this.